index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1 class="logo">Welcome</h1>
+        <a href="#"><img src="images/menu_icon.png" alt="menu" /></a>
+      </header>
+
+      <section class="headline">
+        <h2>
+          Hey there. I’m Mariane <br /><span>I’m a software developer</span>
+        </h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+          <a href="#"
+            ><img src="images/angellist.png" class="social" alt=""
+          /></a>
+          <a href="#"><img src="images/github.png" class="social" alt="" /></a>
+          <a href="#"
+            ><img src="images/Linkedin-icon.png" class="social" alt=""
+          /></a>
+          <a href="#"><img src="images/twitter.png" class="social" alt="" /></a>
+          <a href="#"><img src="images/medium.png" class="social" alt="" /></a>
+        </div>
```

Comment:
- [ ] Please consider using a `ul` tag to wrap all the social icons instead of `div` tag and use `li` tag to wrap each of the social icons. This is because the social icons are unordered lists and therefore should be wrapped with a `ul` and `li` tag and also to improve the code structure.